index.html

```diff
@@ -0,0 +1,17 @@
+<html>
```

Comment:
```suggestion
<html lang="ja">
```
`lang`の属性を必ず指定した方がSEO的に良い